core-services/prow/02_config/rh-ecosystem-edge/NVIDIA-gpu-add-on-init-container/_prowconfig.yaml

```
@@ -0,0 +1,22 @@
+branch-protection:
+  orgs:
+    rh-ecosystem-edge:
+      repos:
+        NVIDIA-gpu-add-on-init-container:
+          protect: false
```

I think you want to add branch protection as well, no?
```suggestion
          branches:
            main:
              protect: true
          protect: false
```

---

Yep -   adding now. Thanks